CefSharp.WinForms/Internals/MovingListener.cs

```diff
@@ -0,0 +1,155 @@
+﻿// Copyright © 2014 The CefSharp Authors. All rights reserved.
+//
+// Use of this source code is governed by a BSD-style license that can be found in the LICENSE file.
+
+using CefSharp.Internals;
+using System;
+using System.Drawing;
+using System.Runtime.InteropServices;
+using System.Windows.Forms;
+
+namespace CefSharp.WinForms.Internals
+{
+    internal class MovingListener : NativeWindow
+    {
+        public event EventHandler<EventArgs> Moving;
+
+        private bool isMoving = false;
+        private Rectangle movingRectangle;
+
+        private Control ParentControl { get; set; }
+
+        private Form ParentForm { get; set; }
+
+        public MovingListener(Control parent)
+        {
+            ParentControl = parent;
```

Comment:
Of `parent/ParentControl` you mean? i.e. rename `ParentControl` to `BrowserControl` and the type from Control to `ChromiumWebBrowser`? or something?

I have no objections if that's what you're asking. Let me know and I'll fold it in.